src/releng_channel_status/tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import pytest
+
+import backend_common
+
+
+# @pytest.fixture(scope='session')
```

Comment:
@garbas may have more input about how to not use a DB, for this project

Reply:
Ah, I have removed database from settings.py and from app extensions. The code was commented because I did not start the tests yet.